api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -217,6 +217,18 @@ paths:
                 value: 'child_signal'
               combination:
                 value: ['parent_signal', 'signal']
+        - name: changes_in_children
+          in: query
+          description: >-
+            Filters all parent signals...
+          required: false
+          schema:
+            type: boolean
+            example:
+              true: true
```

Comment:
Please remove any values other than `true` or `false` from this parameter's documentation --- consistent with the other filter parameters that take a Boolean value. (See also a comment below on `api/app/signals/apps/api/v1/filters/utils.py`.)

Reply:
done